src/cloud-api-adaptor/ibmcloud/ROKS_SETUP.md

```diff
@@ -354,3 +328,86 @@ Otherwise:
     ./caa-provisioner-cli -action=uninstall
     popd
     ```
+
+## Deploy a test Trustee
+
+The following instructions can be used to set up a simple Trustee with an HTTP endpoint for testing attestation.
+
+1. Create a Ubuntu VSI in the same VPC as your ROKS cluster.
+    ```
+    ibmcloud is instance-create "$CLUSTER_NAME-trustee" "$VPC_ID" "$ZONE" "bx2-2x8" "$SUBNET_ID" --image "r014-85b1a9ec-369b-41d6-b921-39666d4139d1" --keys "$SSH_KEY_ID" --allow-ip-spoofing false
+    ```
+
+1. SSH into the new VSI and run the following commands.
+    > Tip: you can SSH to the private IP of the VSI from a pod or node in your ROKS cluster, since they are running in the same VPC.
+
+    First, run the following commands to install docker and oras:
+    ```
+    sudo apt-get update
+    sudo apt-get install ca-certificates curl
+    sudo install -m 0755 -d /etc/apt/keyrings
+    sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+    sudo chmod a+r /etc/apt/keyrings/docker.asc
+    echo \
+      "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+      $(. /etc/os-release && echo "${UBUNTU_CODENAME:-$VERSION_CODENAME}") stable" | \
+      sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    sudo apt-get update
+    sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin -y
+    sudo snap install oras --classic
+    ```
+
+    Next, deploy the Trustee services:
+    ```
+    git clone https://github.com/confidential-containers/trustee.git
+    cd trustee
+    openssl genpkey -algorithm ed25519 > kbs/config/private.key
+    openssl pkey -in kbs/config/private.key -pubout -out kbs/config/public.pub
+    sudo docker compose up -d
+    ```
+
+    Finally, configure an example key that can be used for CDH testing:
+    ```
+    oras pull ghcr.io/confidential-containers/staged-images/kbs-client:latest
+    chmod +x kbs-client
+    cat > kbsres1_key1 << EOF
+    res1val1
+    EOF
+    ./kbs-client --url http://127.0.0.1:8080 config --auth-private-key kbs/config/private.key set-resource --resource-file kbsres1_key1 --path default/kbsres1/key1
+    ```
+
+1. Configure your confidential containers environment to use the Trustee.
+
+    First, use the VSI IP to set the `KBS_SERVICE_ENDPOINT` environment variable to the URL of the Trustee:
+    ```
+    export KBS_SERVICE_ENDPOINT="http://$(ibmcloud is instance "$CLUSTER_NAME-trustee" --output JSON | jq -r '.network_interfaces[].primary_ip.address'):8080"
+    ```
+
+    Then, configure the Trustee in your `peerpods-cluster.properties` file by  running the following commands:
+    ```
+    export INITDATA=$(cat <<EOF | base64
+    algorithm = "sha256"
+    version = "0.1.0"
+
+    [data]
+    "aa.toml" = '''
+    [token_configs]
+    [token_configs.coco_as]
+    url = "$KBS_SERVICE_ENDPOINT"
```

Comment:
I think this section is no longer necessary given https://github.com/confidential-containers/guest-components/pull/980

Reply:
Thanks, I'll wait for it to definitely be there and then simplify the config in a followup PR.